src/java/fr/paris/lutece/plugins/forms/service/FormService.java

```diff
@@ -604,4 +606,23 @@ public void saveOrUpdateField( Entry entry, String fieldName, String title, Stri
             FieldHome.update( GenericAttributesUtils.createOrUpdateField( entry, fieldName, title, value ) );
         }
     }
+
+    /**
+     * Get the responses which verify the filter
+     * 
+     * @param responseFilter
+     *            the filter
+     * @return the list of responses
+     */
+    public List<Response> getResponseList( ResponseFilter responseFilter )
```

Comment:
unnecessary method

Reply:
The change requested has been implemented